js/code1.js

```diff
@@ -0,0 +1,18 @@
+function code(){
+    let n = prompt("enter a number n :");
+    arr = [];
+    for (let i = 0; i < n; i++) {
+        arr.push(parseInt(Math.random()*20));
+    }
+    arr.sort(function(a, b) {
+    return a - b;
+    });
+    alert(arr);
+    console.log(arr);
+    let set = [...new Set(arr)]
+    alert(set);

```

Comment:
This will display the contents of the set. Not the count of distinct elements. `set.size` will display the count of distinct elements.